guides/doc-Managing_Organizations_and_Locations/master.adoc

```diff
@@ -0,0 +1,18 @@
+= Managing Organizations and Locations in {Project}
```

Comment:
```suggestion
= Managing Organizations and Locations in {Project}
```
I think this title needs to be moved down as it is building before the atttributes are being invoked. It renders as {Project} on build.

Reply:
Fixed.